### PR TITLE
OCP 4.15 Release Notes for P/Z

### PR DESCRIPTION
**Pull request by alishaIBM**
Version(s): 4.15

Issue: 
- https://issues.redhat.com/browse/MULTIARCH-3999

Link to docs preview:
- P: https://70664--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-ibm-power
- Z: https://70664--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-ibm-z

Draft link: 
- https://docs.google.com/document/d/1PH_dTQ-_9NJlzgK_ycGehQSToWUDoXnqb1kOMeb214M/edit

QE review:
- For P : Manoj Kumar, Luvella , Julie
- for Z : Holger Wolf, Florian Leber
- [x] QE has approved this change.

Branch : enterprise-4.15
@SNiemann15

### CODE REVIEW

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -278,6 +344,10 @@ For more information, see xref:../installing/installing_ibm_cloud_public/user-ma
 |====
 |Feature |{ibm-power-name} |{ibm-z-name} and {ibm-linuxone-name}
 
+|cert-manager Operator for Red Hat OpenShift
```
Comment (SNiemann15) [inline release_notes/ocp-4-15-release-notes.adoc:347]:
```suggestion
|{cert-manager-operator}
```

File: release_notes/ocp-4-15-release-notes.adoc [hunk 2]
```diff
@@ -75,16 +75,70 @@ For more information, see xref:../installing/installing_ibm_cloud_public/user-ma
...
+* cert-manager Operator for Red Hat OpenShift
+* {ibm-power-title} Virtual Server Block CSI Driver Operator
+* Installer-provisioned Infrastructure Enablement for {ibm-power-title} Virtual Server
+* nx-gzip for Power (Hardware Acceleration)
+* openshift-install to support various SMT levels on Power (Hardware Acceleration)
+
 [discrete]
 === {ibm-power-title}, {ibm-z-title}, and {ibm-linuxone-title} support matrix
 
+Starting in {product-title} 4.15, Extended Update Support (EUS) is extended to the {ibm-power-title} and the {ibm-z-title} platform.  For more information, see the link:https://access.redhat.com/support/policy/updates/openshift-eus[OpenShift EUS Overview].
```
Comment (SNiemann15) [inline release_notes/ocp-4-15-release-notes.adoc:140]:
```suggestion
Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to the {ibm-power-title} and the {ibm-z-title} platform.  For more information, see the link:https://access.redhat.com/support/policy/updates/openshift-eus[OpenShift EUS Overview].
```

**Review by SNiemann15 — CHANGES_REQUESTED** [review]
@alishaIBM Only use the `{ibm-xxx-title}` attributes in titles or when referencing to titles. Otherwise use: `{ibm-xxx-name} `
I've left suggestions for IBM Z apply that to the Power part too.

File: release_notes/ocp-4-15-release-notes.adoc [hunk 3]
```diff
@@ -80,16 +80,68 @@ For more information, see xref:../installing/installing_ibm_cloud_public/user-ma
...
 
+The {ibm-power-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components.
+
+This release introduces support for the following features on {ibm-power-name}:
+
+* Agent-based Installer
+* {cert-manager-operator}
+* {ibm-power-name} Virtual Server Block CSI Driver Operator
+* Installer-provisioned Infrastructure Enablement for {ibm-power-name} Virtual Server
+* Multi-architecture Control Plane with support of Intel and Power workers
```
Comment (prb112) [inline release_notes/ocp-4-15-release-notes.adoc:136]:
```suggestion
* Multi-architecture Power Control Plane with support of Intel and Power workers
```

Comment (alishaIBM) [reply release_notes/ocp-4-15-release-notes.adoc]: Updated the PR. Thanks @prb112

**Review by prb112 — APPROVED** [review]

File: release_notes/ocp-4-15-release-notes.adoc [hunk 4]
```diff
@@ -80,16 +80,68 @@ For more information, see xref:../installing/installing_ibm_cloud_public/user-ma
...
+* {cert-manager-operator}
+
+[discrete]
+==== Installing a cluster with an LPAR on {ibm-z-title} and {ibm-linuxone-title}
+
+{product-title} now supports user-provisioned installation of {product-title} 4.15 with a logical partition (LPAR) on {ibm-z-title} and {ibm-linuxone-title}
+
+For installation instructions, see the following documentation:
+
+* xref:../installing/installing_ibm_z/installing-ibm-z-kvm.html#installing-rhcos-using-ibm-secure-execution_installing-ibm-z-kvm[Installing a cluster with an LPAR on {ibm-z-name} and {ibm-linuxone-name}]
```
Comment (SNiemann15) [inline release_notes/ocp-4-15-release-notes.adoc:109]:
```suggestion
* xref:../installing/installing_ibm_z/installing-ibm-z-lpar.adoc#installing-ibm-z-lpar[Installing a cluster in an LPAR on {ibm-z-name} and {ibm-linuxone-name}]
* xref:../installing/installing_ibm_z/installing-restricted-networks-ibm-z-lpar.adoc#installing-restricted-networks-ibm-z-lpar[Installing a cluster in an LPAR on {ibm-z-name} and {ibm-linuxone-name} in a restricted network]

```

File: release_notes/ocp-4-15-release-notes.adoc [hunk 5]
```diff
@@ -80,16 +80,68 @@ For more information, see xref:../installing/installing_ibm_cloud_public/user-ma
...
+//update above with the correct link when the PR is merged.
```
Comment (SNiemann15) [inline release_notes/ocp-4-15-release-notes.adoc:110]:
```suggestion

```

File: release_notes/ocp-4-15-release-notes.adoc [hunk 6]
```diff
@@ -80,16 +80,68 @@ For more information, see xref:../installing/installing_ibm_cloud_public/user-ma
...
+
+[discrete]
+==== {ibm-z-title} and {ibm-linuxone-title} notable enhancements
+
+The {ibm-z-name} and {ibm-linuxone-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components and concepts.
+
+This release introduces support for the following features on {ibm-z-name} and {ibm-linuxone-name}:
+
+* Agent-based Installer
+* {cert-manager-operator}
```
Comment (SNiemann15) [inline release_notes/ocp-4-15-release-notes.adoc:100]:
```suggestion
* {cert-manager-operator}
* `s390x` control plane with `x86_64` multi-architecture compute nodes
```

**Comment by holgwolf** [comment]
LGTM

File: release_notes/ocp-4-15-release-notes.adoc [hunk 7]
```diff
@@ -129,16 +129,70 @@ You can now move etcd from a root volume (Cinder) to a dedicated ephemeral local
...
+
+This release introduces support for the following features on {ibm-power-name}:
+
+* Agent-based Installer
+* {cert-manager-operator}
+* {ibm-power-name} Virtual Server Block CSI Driver Operator
+* Installer-provisioned Infrastructure Enablement for {ibm-power-name} Virtual Server
+* Multi-architecture Power Control Plane with support of Intel and {ibm-power-name} workers
+* nx-gzip for Power10 (Hardware Acceleration)
+* openshift-install to support various SMT levels on {ibm-power-name} (Hardware Acceleration)
```
Comment (alishaIBM) [inline release_notes/ocp-4-15-release-notes.adoc:189]: @prb112 request you to review above component names once again. The update is I have added {ibm-power-name}.

Comment (prb112) [reply release_notes/ocp-4-15-release-notes.adoc:189]: Looks good.

Comment (prb112) [reply release_notes/ocp-4-15-release-notes.adoc:189]: /lgtm

**Comment by alishaIBM** [comment]
/label peer-review-needed

fyi: QE approval pending.

File: release_notes/ocp-4-15-release-notes.adoc [hunk 8]
```diff
@@ -129,16 +129,70 @@ You can now move etcd from a root volume (Cinder) to a dedicated ephemeral local
 
 [id="ocp-4-15-ibm-z"]
 === {ibm-z-title} and {ibm-linuxone-title}
+With this release, {ibm-z-name} and {ibm-linuxone-name} are now compatible with {product-title} {product-version}. The installation can be performed with z/VM, LPAR, or {op-system-base-full} Kernel-based Virtual Machine (KVM). For installation instructions, see the following documentation:
+
+* xref:../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing-to-install-on-ibm-z[Installing a cluster with on {ibm-z-title} and {ibm-linuxone-title}]
+
+[IMPORTANT]
+====
+Compute nodes must run {op-system-first}
```
Comment (xenolinux) [inline release_notes/ocp-4-15-release-notes.adoc:138]:
```suggestion
Compute nodes must run {op-system-first}.
```

File: release_notes/ocp-4-15-release-notes.adoc [hunk 9]
```diff
@@ -129,16 +129,70 @@ You can now move etcd from a root volume (Cinder) to a dedicated ephemeral local
...
 === {ibm-power-title}
 
+{ibm-power-name} is now compatible with {product-title} {product-version}. For installation instructions, see the following documentation:
+
+* xref:../installing/installing_ibm_power/installing-ibm-power.adoc#installing-ibm-power_installing-ibm-power[Installing a cluster on {ibm-power-name}]
+* xref:../installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc#installing-restricted-networks-ibm-power_installing-restricted-networks-ibm-power[Installing a cluster on {ibm-power-name} in a restricted network]
+
+[IMPORTANT]
+====
+Compute nodes must run {op-system-first}
```
Comment (xenolinux) [inline release_notes/ocp-4-15-release-notes.adoc:173]:
```suggestion
Compute nodes must run {op-system-first}.
```

File: release_notes/ocp-4-15-release-notes.adoc [hunk 10]
```diff
@@ -129,16 +129,70 @@ You can now move etcd from a root volume (Cinder) to a dedicated ephemeral local
 
 [id="ocp-4-15-ibm-z"]
 === {ibm-z-title} and {ibm-linuxone-title}
+With this release, {ibm-z-name} and {ibm-linuxone-name} are now compatible with {product-title} {product-version}. The installation can be performed with z/VM, LPAR, or {op-system-base-full} Kernel-based Virtual Machine (KVM). For installation instructions, see the following documentation:
```
Comment (xenolinux) [inline release_notes/ocp-4-15-release-notes.adoc:132]:
```suggestion
With this release, {ibm-z-name} and {ibm-linuxone-name} are now compatible with {product-title} {product-version}. You can perform the installation with z/VM, LPAR, or {op-system-base-full} Kernel-based Virtual Machine (KVM). For installation instructions, see the following documentation:
```

File: release_notes/ocp-4-15-release-notes.adoc [hunk 11]
```diff
@@ -129,16 +129,70 @@ You can now move etcd from a root volume (Cinder) to a dedicated ephemeral local
...
+This release introduces support for the following features on {ibm-z-name} and {ibm-linuxone-name}:
+
+* Agent-based Installer
+* {cert-manager-operator}
+* `s390x` control plane with `x86_64` multi-architecture compute nodes
+
+[discrete]
+==== Installing a cluster with an LPAR on {ibm-z-title} and {ibm-linuxone-title}
+
+{product-title} now supports user-provisioned installation of {product-title} 4.15 with a logical partition (LPAR) on {ibm-z-title} and {ibm-linuxone-title}
```
Comment (xenolinux) [inline release_notes/ocp-4-15-release-notes.adoc:155]:
```suggestion
{product-title} now supports user-provisioned installation of {product-title} 4.15 with a logical partition (LPAR) on {ibm-z-title} and {ibm-linuxone-title}.
```

**Review by xenolinux — COMMENTED** [review]
A few minor comments; otherwise LGTM!

/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

**Comment by manojnkumar** [comment]
LGTM

**Comment by alishaIBM** [comment]
/label merge-review-needed

fyi: QE has approved this change.

File: release_notes/ocp-4-15-release-notes.adoc [hunk 12]
```diff
@@ -129,16 +129,70 @@ You can now move etcd from a root volume (Cinder) to a dedicated ephemeral local
...
+
+This release introduces support for the following features on {ibm-power-name}:
+
+* Agent-based Installer
+* {cert-manager-operator}
+* {ibm-power-name} Virtual Server Block CSI Driver Operator
+* Installer-provisioned Infrastructure Enablement for {ibm-power-name} Virtual Server
+* Multi-architecture Power Control Plane with support of Intel and {ibm-power-name} workers
+* nx-gzip for Power10 (Hardware Acceleration)
+* openshift-install to support various SMT levels on {ibm-power-name} (Hardware Acceleration)
```
Comment (adellape) [inline release_notes/ocp-4-15-release-notes.adoc:189]: Markup `openshift-install` and add something like utility/command/binary/tool after:
```suggestion
* The `openshift-install` utility to support various SMT levels on {ibm-power-name} (Hardware Acceleration)
```
I couldn't find a definitive answer for what to call it, but those were some options from around the repo.

**Review by adellape — COMMENTED** [review]
Just one comment I'd like to hold merge on. Please re-queue for merge after addressing. Thank you!

**Comment by alishaIBM** [comment]
/label merge-review-needed

> Just one comment I'd like to hold merge on. Please re-queue for merge after addressing. Thank you!

Addressed the comments. Re-queuing for merge. Thanks. FYI: @adellape

**Review by JoeAldinger — APPROVED** [review]
One question for future consideration. lgtm

File: release_notes/ocp-4-15-release-notes.adoc [hunk 13]
```diff
@@ -266,15 +328,19 @@ For more information, see link:https://docs.openshift.com/builds/1.0/about/overv
...
+|Hosted control planes (Technology Preview)
+|Supported
+|Supported
+
 |IBM Secure Execution
 |Unsupported
 |Supported
 
-|{ibm-power-server-name} Block CSI Driver Operator (Technology Preview)
+|{ibm-power-server-name} Block CSI Driver Operator
```
[the remark this answers is not in the document]
Comment (SNiemann15) [reply release_notes/ocp-4-15-release-notes.adoc:339]: @JoeAldinger If you look at the table headers you can see that column 2 is IBM Power, column 3 is IBM Z. 
The two features are supported on IBM Power but not on IBM Z and the plan is to move from TP to GA with 4.15.

[the remark this answers is not in the document]
Comment (JoeAldinger) [reply release_notes/ocp-4-15-release-notes.adoc:339]: Sorry, my confusion.

**Pull request merged**